### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by mariusmihaic**
- Rebranding to multiversx
- Also re-generated all proto files

### CODE REVIEW

**Review by AdoAdoAdo — APPROVED** [review]

**Review by schimih — APPROVED** [review]

File: core/libLocator/vmLocator_windows.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+//go:build windows
```
Comment (miiu96) [inline core/libLocator/vmLocator_windows.go:1]: why did you add this line?

File: core/constants.go [hunk 2]
```diff
@@ -175,17 +175,17 @@ const SemiFungibleESDT = "SemiFungibleESDT"
...
+// RelayedTransactionV2 is the key for the optimized electronic meta/gassless/relayed transaction standard
 const RelayedTransactionV2 = "relayedTxV2"
 
 // SCDeployInitFunctionName is the key for the function which is called at smart contract deploy time
 const SCDeployInitFunctionName = "_init"
 
-// ElrondProtectedKeyPrefix is the key prefix which is protected from writing in the trie - only for special builtin functions
-const ElrondProtectedKeyPrefix = "ELROND"
+// ProtectedKeyPrefix is the key prefix which is protected from writing in the trie - only for special builtin functions
+const ProtectedKeyPrefix = "electronic"
```
Comment (iulianpascalau) [inline core/constants.go]: please revert this

File: core/libLocator/vmLocator_windows.go (hunk 1 above)
Comment (iulianpascalau) [reply core/libLocator/vmLocator_windows.go:1]: the new format caused by the IDE, I guess

**Review by iulianpascalau — CHANGES_REQUESTED** [review]

File: core/constants.go [hunk 3]
```diff
@@ -185,7 +185,7 @@ const RelayedTransactionV2 = "relayedTxV2"
 const SCDeployInitFunctionName = "_init"
 
 // ProtectedKeyPrefix is the key prefix which is protected from writing in the trie - only for special builtin functions
-const ProtectedKeyPrefix = "electronic"
+const ProtectedKeyPrefix = "ELROND"
```
Comment (iulianpascalau) [inline core/constants.go:188]: perhaps:
```GO
const ProtectedKeyPrefix = "E" + "L" + "R" + "O" + "N" + "D"
```

and add a unit test:

Comment (mariusmihaic) [reply core/constants.go:188]: Changed it as suggested

**Review by iulianpascalau — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]
